app/components/Header/Header.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import AppBar from '@material-ui/core/AppBar';
```

Comment:
Next time let's move the imports that starts with `@` like this
```

import React, { Component } from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import AccountCircleIcon from '@material-ui/icons/AccountCircle';
import MenuIcon from '@material-ui/icons/Menu';
```

So they can be classified as "grouped" ones

Reply:
Noted.